components/gitpod-cli/cmd/ssh.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/gitpod-io/gitpod/gitpod-cli/pkg/gitpod"
+
+	"github.com/spf13/cobra"
+)
+
+var gitpodHost = os.Getenv("GITPOD_HOST")
```

Comment:
This is pretty ugly, but it's the best I figured for dynamically filling the Gitpod host in the help menu.

Reply:
ok, but it can break in future

Reply:
add a comment here that is a hack and should not be used in other places, that we should use info endpoint instead with TODO to remove